app/src/main/java/net/gsantner/markor/activity/DocumentEditFragment.java

```diff
@@ -255,50 +269,27 @@ public void onFragmentFirstTimeVisible() {
         StringUtils.setSelectionAndShow(_hlEditor, startPos);
     }
 
-    public void resume() {
-        loadDocument();
-        if (_hlEditor != null) {
-            _hlEditor.onResume();
-        }
-        if (_webView != null) {
-            _webView.onResume();
-        }
-    }
-
     @Override
     public void onResume() {
-        resume();
+        loadDocument();
+        _webView.onResume();
         super.onResume();
```

Comment:
Removed lots of tests. We should not encounter a silent failure to write the file now. (There is a greater chance of npe, looking into how we can catch this and save the file contents)